api/src/main/resources/messages.properties

```diff
@@ -642,7 +642,7 @@ coreapps.conditionui.updateCondition.success=Condition Saved Successfully
 coreapps.conditionui.updateCondition.error=Error Saving condition
 coreapps.conditionui.removeCondition=Remove Condition
 coreapps.conditionui.removeCondition.message=Are you sure you want to remove {0} condition for this patient?
-coreapps.conditionui.editCondition=Edit Condition: {0}
```

Comment:
This could be used by other places. So i do not feel comfortable merging it together with changes for editing a condition.

Reply:
do you suggest to have another property for "Edit Condition" string or to display the condition next to the heading ?

Reply:
What happens if you start by just leaving as it was?

Reply:
The Heading would be displayed as "Edit Condition: {0}" on the page

Reply:
I passed an empty string as a parameter to {0} as a result of which the heading is being displayed as desired without having the change the format of the strings in properties file. Hope there are no other issues for the PR to be merged :)